nginx/ng/map.jinja

```diff
@@ -145,7 +145,9 @@
         'disabled_postfix': '.disabled',
         'symlink_opts': {},
         'rename_opts': {},
-        'managed_opts': {},
+        'managed_opts': {
+            'makedirs': True,
```

Comment:
This is not required if you have `{{ conf_state_id }}` require the directory to exist already.

Reply:
You can now place conf files outside `sites-available`/`sites-enabled` that's why I added this. Check #172 #176.

Reply:
Checking current formula code, I do not see the relation between extra_config files and `servers_config.sls` configuration files. Also, for reading the two links you provided, it does not seem that `managed_opts` is being used there.

Reply:
Sorry for coming back 4 months after you review, just resumed my work on this. `managed_opts` is used in https://github.com/saltstack-formulas/nginx-formula/blob/master/nginx/ng/servers_config.sls#L111 and in #176 I have put an example like this: